### PR TITLE
Aida/comp 89

### PR DESCRIPTION
**Pull request by aidacoughlan**
I've changed the name of the website, please feel free to comment with any other name suggestions.

### CODE REVIEW

File: software_engineer.rar
Comment (AprilPolubiec) [inline software_engineer.rar:1]: I dont think this is meant to be here

File: web/static/index.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ function renderStation(station, availability, markers, markerBounds, map) {
     map,
     title: station.Name,
     label: (availability.ElectricBikesAvailable + availability.MechanicalBikesAvailable).toString(),
+    icon: {
+      url: 'http://127.0.0.1:5000/static/marker-icon.webp',
```
Comment (AprilPolubiec) [inline web/static/index.js:52]: i'm not exactly sure if this is going to work in production... 🤔 we can give it a try but before we do, have you tried just linking to the file directly? Like "/static/marker-icon"?

File: web/static/marker-icon.webp
Comment (AprilPolubiec) [inline web/static/marker-icon.webp:1]: did you try converting this to a png or jpg? I wonder if the `webp` was causing problems

File: software_engineer.rar
Comment (aidacoughlan) [reply software_engineer.rar:1]: You're_ right, I will remove this!

File: web/static/index.js (hunk 1 above)
Comment (aidacoughlan) [reply web/static/index.js:52]: I'll give that a go and try that and let you know

File: web/static/marker-icon.webp
Comment (aidacoughlan) [reply web/static/marker-icon.webp:1]: The png was giving me problems and I looked the web up and apparently, it's meant to be better quality but I will try a png or jpg and see how it loads and looks etc. Also, I didn't mean to open a pr for the marker. I created a separate branch and just changed the name and that's what this pr is for. I somehow combined the branches :/

**Pull request closed without merging**